build.sbt

```diff
@@ -11,24 +9,13 @@ scalaVersion := "2.13.1"
 
 lazy val toolVersionKey = SettingKey[String]("the version of the underlying tool retrieved from patterns.json")
 
-toolVersionKey := {
-  case class Patterns(name: String, version: String)
-  implicit val patternsIso: IsoLList[Patterns] =
-    LList.isoCurried((p: Patterns) => ("name", p.name) :*: ("version", p.version) :*: LNil) {
-      case (_, n) :*: (_, v) :*: LNil => Patterns(n, v)
-    }
-
-  val jsonFile = (resourceDirectory in Compile).value / "docs" / "patterns.json"
-  val json = Parser.parseFromFile(jsonFile)
-  val patterns = json.flatMap(Converter.fromJson[Patterns])
-  patterns.get.version
-}
+toolVersionKey := "6.21.0"
 
 libraryDependencies ++= {
   val toolVersion = toolVersionKey.value
```

Comment:
You can raname `toolVersionKey` to `toolVersion` and use it directly without saving it.